spec/unit/bits_service_client/bits_service_client_spec.rb

```diff
@@ -454,6 +454,26 @@
 
       request = stub_request(:put, private_resource_endpoint).to_return(status: 201, body: "{\"sha1\":\"abc\", \"sha256\":\"def\"}")
 
+      subject.cp_to_blobstore(file_path, key)
+      expect(request).to have_been_requested
+    end
+  end
+  context 'Logging' do
```

Comment:
This whole added block seems to be a duplicate of the code above. Was it added by accident?

Reply:
this is gone with
ca5f427